main.css

```diff
@@ -162,6 +162,11 @@ body {
     text-align: center;
 }
 
+.landing-main-link:hover {
+    box-shadow: 0px 0px 8px 0.5px grey;
```

Comment:
use variable for color

Reply:
Ok, actually it was only one small thing not planning to repeat anywhere so i skipped variable